CoherenceSpectrogramViewer/Source/CoherenceVisualizer.cpp

```diff
@@ -546,6 +546,7 @@ void CoherenceVisualizer::refresh()
 				String Idchn = "#" + std::to_string(k + 1);
 				plotHoldingVect[i]->setTitle("Power vs Frequency: CH" + Idchn);
 				plotHoldingVect[i]->plotxy(XYline(freqStart, freqStep, processor->ttlpwr[i], 1, Colours::yellow));
+                plotHoldingVect[i]->setAutoRescale(true);

```

Comment:
Kind of hacky way, but telling it to rescale every time after plotting the line makes the graph scale correctly.